### DISCUSSION TITLE
Where can I find logs for this app

### DISCUSSION THREAD

**Discussion by ChibyX** [mjakeman/extension-manager#384] [Q&A]
I have a problem with app, so I would like to find out what causes it

**Comment by oscfdezdz** [comment]
Running it from the terminal should give you some details about crashes.

Flatpak: `flatpak run com.mattjakeman.ExtensionManager`

Traditional package: `extension-manager`

For more advanced logging you can use gdb:

For Flatpak, check https://docs.flatpak.org/en/latest/debugging.html

For traditional package formats, install `gdb` and run: `gdb extension-manager`, type `run` and when the crash or bug occurs, `bt`.
